mender-convert-package

```diff
@@ -250,9 +250,9 @@ else
   run_and_log_cmd "${PARTED} -s ${img_path} unit s mkpart primary fat32 ${boot_part_start} ${boot_part_end}"
 fi
 run_and_log_cmd "${PARTED} -s ${img_path} set 1 boot on"
-run_and_log_cmd "${PARTED} -s ${img_path} -- unit s mkpart primary ext2 ${rootfsa_start} ${rootfsa_end}"
-run_and_log_cmd "${PARTED} -s ${img_path} -- unit s mkpart primary ext2 ${rootfsb_start} ${rootfsb_end}"
-run_and_log_cmd "${PARTED} -s ${img_path} -- unit s mkpart primary ext2 ${data_start} ${data_end}"
+run_and_log_cmd "${PARTED} -s ${img_path} -- unit s mkpart primary ext4 ${rootfsa_start} ${rootfsa_end}"
+run_and_log_cmd "${PARTED} -s ${img_path} -- unit s mkpart primary ext4 ${rootfsb_start} ${rootfsb_end}"
+run_and_log_cmd "${PARTED} -s ${img_path} -- unit s mkpart primary ext4 ${data_start} ${data_end}"
```

Comment:
I do not know why `parted` requires the fstype option here and what it does with it. This just creates the partitions.

According to the [parted man page](https://www.gnu.org/software/parted/manual/html_node/mkpart.html),

```
fs-type must be one of these supported file systems:

ext2
fat16, fat32
hfs, hfs+, hfsx
linux-swap
NTFS
reiserfs
ufs
btrfs
```

But again I do not really see how these values have any impact on anything.

Reply:
+1